po/pt_BR.po

```diff
@@ -1,5 +1,664 @@
 msgid ""
 msgstr ""
+"Language: pt_BR\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
-"Plural-Forms: nplurals=2; plural=n>1;\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: codedracula <62944284+codedracula4ec@users.noreply."
+"github.com>\n"
+"Language-Team: \n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.1\n"
+
+#: Dialogs/ExportDialog.vala:151
+msgid "Export to:"
+msgstr "Exportar para:"
+
+#: Dialogs/ExportDialog.vala:157
+msgid "Select Folder"
+msgstr "Selecionar pasta"
+
+#: Dialogs/ExportDialog.vala:166
+msgid "Format:"
+msgstr "Formato:"
+
+#: Dialogs/ExportDialog.vala:180
+msgid "Quality:"
+msgstr "Qualidade:"
+
+#: Dialogs/ExportDialog.vala:192
+msgid "Compression:"
+msgstr "Compressão:"
+
+#: Dialogs/ExportDialog.vala:206
+msgid "Transparency:"
+msgstr "Transparência:"
+
+#: Dialogs/ExportDialog.vala:219
+msgid "Scale:"
+msgstr "Escala:"
+
+#: Dialogs/ExportDialog.vala:243 FileFormat/FileManager.vala:49
+#: FileFormat/FileManager.vala:105
+msgid "Cancel"
+msgstr "Cancelar"
+
+#: Dialogs/ExportDialog.vala:250 Dialogs/ShortcutsDialog.vala:63
+#: Layouts/HeaderBar.vala:112
+msgid "Export"
+msgstr "Exportar"
+
+#: Dialogs/ReleaseDialog.vala:47 Dialogs/SettingsDialog.vala:217
+msgid ""
+"WARNING!\n"
+"Akira is still under development and not ready for production. Missing "
+"features, random bugs, and black holes opening in your kitchen are to be "
+"expected."
+msgstr ""
+"ATENÇÃO!\n"
+"Akira está sob desenvolvimento e não está pronto para produção. É de se "
+"esperar mal funcionamento e inexistência de funções."
```

Comment:
É de se esperar o mau funcionamento e a inexistência de algumas funcionalidades.

Reply:
That makes sense. Tanks!